annotations/builder/src/main/java/io/sundr/builder/internal/BuilderContext.java

```diff
@@ -1077,66 +1074,8 @@ public BuilderContext(Elements elements, Types types, Boolean generateBuilderPac
 
         .addNewMethod()
         .withNewModifiers().withPublic().endModifiers()
-        .withName("accept")
```

Comment:
While we do drop this large chunk of boilerplate and I am grateful for it, I cannot help wonder what exactly allow us to do so.

I understand that we moved the target methods from BaseFluent to Visitable, but don't we need to move their definition too? In other words  should we construct a Visitable TypeDef that includes these `accept` methods?

Reply:
The Visitable TypeDef already has the necessary logic to support the accept default methods.  The only new method is getVisitableMap, which handling was added for on both Visitable and BaseFluent.  The one bit of messiness there is that it has to be added to Visitable after VisitableMap is defined - is there a better way to do that?